CodeStarterCamp_Week2/main.swift

```diff
@@ -6,7 +6,28 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
 
-print("Hello, World!")
+func makeLottoNumbers() -> Set<Int> {
+    var result: Set<Int> = Set<Int>()
+    
+    while result.count != 6 {
+        result.insert(Int.random(in: 1...45))
+    }
+    return result
+}
+
+func checkLottoNumbers(myNumbers: Set<Int>, matching: Set<Int>) {
+    let result: Set<Int> = myNumbers.intersection(matching)
+    
+    if result.count == 0 {
+        print("아쉽지만 겹치는 번호가 없습니다.")
+    } else {
+        let sameNumbers: String = result.map {String($0)}.joined(seperator: ", ")
```

Comment:
```suggestion
    let sameNumbers: String = result.map(String.init).joined(separator: ", ")
```
요렇게 쓰면 더 읽기 쉬울거 같아요!